Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>.idea/\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-16LE
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision 79abb74f1eb8407447951fb6d5dabf7a37e6af30)
+++ b/.gitignore	(date 1753145092759)
@@ -1,1 +1,1 @@
-.idea/
+#.idea/
